templates/default/jobwatcher.cfg.erb

```diff
@@ -4,3 +4,4 @@ scheduler = <%= node['cfncluster']['cfn_scheduler'] %>
 stack_name = <%= node['cfncluster']['stack_name'] %>
 cfncluster_dir = <%= node['cfncluster']['base_dir'] %>
 proxy = <%= node['cfncluster']['cfn_proxy'] %>
+instance_types_data = <%= node['instance_types_data'].to_json %>
```

Comment:
why do we need the to_json? the node is reading this as a string. what is the to_json for?

Reply:
Because it is a Hash object in memory, so it would get serialized like here:
```
{:key=>"key", :value=>"value"}
```

Reply:
Thanks